templates/store/_entity-header.html

```diff
@@ -50,7 +50,7 @@ <h1 itemprop="name" class="u-no-margin--bottom entity__title">
       {% if context.entity.series %}
       Supports:
         {% for supported_release in context.entity.series %}
-          <span class="entity__supported-release {{ supported_release }}">
+          <span class="series-tag series-tag--{{ supported_release }}">
```

Comment:
Don't need the `series-tag` now

Reply:
We do need it as only kubernetes is different so we need a default style to apply first.